external/easylogging++/easylogging++.cc

```diff
@@ -713,9 +713,8 @@ Logger::Logger(const std::string& id, const Configurations& configurations,
 }
 
 Logger::Logger(const Logger& logger) {
-  base::utils::safeDelete(m_typedConfigurations);
   m_id = logger.m_id;
-  m_typedConfigurations = logger.m_typedConfigurations;
+  m_typedConfigurations = logger.m_typedConfigurations ? new base::TypedConfigurations(*logger.m_typedConfigurations) : NULL;
```

Comment:
I think this should be `nullptr`, not `NULL`.

Reply:
Explain the bug ?

Reply:
No "bug" in this instance, it's just deprecated since C++11 and shouldn't be used in new code.